tests/python/rest_api/test_projects.py

```diff
@@ -310,6 +310,27 @@ def test_org_owner_can_get_project_backup(
 
         self._test_can_get_project_backup(user["username"], project["id"])
 
+    def test_can_get_backup_project_when_some_tasks_have_no_data(self, projects):
+        project = next((p for p in projects if 0 < p["tasks"]["count"]))
+
+        # add empty task to project
+        post_method("admin1", "tasks", {"name": "empty_task", "project_id": project["id"]})
```

Comment:
Probably, the response should be checked, as if the request fails, the test won't fail.

Reply:
Added response status check